shortcut_composer/api_krita/enums/action.py

```diff
@@ -0,0 +1,581 @@
+# SPDX-FileCopyrightText: © 2022 Wojciech Trybus <wojtryb@gmail.com>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from krita import Krita as Api
+from enum import Enum
+
+from PyQt5.QtGui import QIcon
+
+
+class Action(Enum):
+    """
+    Contains all known actions of Krita.
+
+    Example usage: `Tool.FREEHAND_BRUSH`
+
+    Available actions:
+    - `TBA`
+    """
+
+    # Filters
```

Comment:
Ja bym to chyba chciał powydzielać do różnych klas Jeszcze nie mam pomysłu na jakie dokładnie grupy - może wystarczą tylko 3 (filtry, akcje, rzadkie), a może zrobić to bardzo szczegółowo (akcje dokumentu, widoku, stosu warstw...)

Tam oryginalnie jest straszny sajgon, bo ich jest w cholerę (jak wiesz) i fajnie by było zaprowadzić jakiś porządek już na tym etapie.

Z kontrolerem Ci się to zepnie, bo typing będzie taki i niezalzeżnie który enum się trafi, będzie działać:
```python
action = Union[Filters, EditActions, ViewActions...]
class ActionController:
    def set_value(value: action) -> None: ...
    def get_label(value: action) -> QIcon: ...
```

Na froncie zawsze można je połączyć (zrzucić je do jednego comboboxa), ale tak masz nad tym jakąś kontrolę, możesz je sortować enumami, grupować, filtrować itd

Plus wtedy można np wywalić przedrostek KRITA_FILTER z takiego  KRITA_FILTER_BLUR i zostaje Ci ładne Filter.BLUR

Reply:
Aye kapitanie, tak też zrobię. Dzięki!